glean-core/src/metrics/labeled.rs

```diff
@@ -67,8 +68,27 @@ where
         }
     }
 
-    fn dynamic_label<'a>(&mut self, label: &'a str) -> &'a str {
-        // TODO(bug 1554970): Fetch seen_labels from the database if empty
+    fn dynamic_label<'a>(&mut self, glean: &Glean, label: &'a str) -> &'a str {
```

Comment:
I know this isn't a public function, but can you please add documentation for what this is supposed to do?